CHANGELOG.md

```diff
@@ -14,6 +14,9 @@
    option to prefer channels which afford better privacy when routing (#1555).
  * `ProbabilisticScorer` now provides access to its estimated liquidity range
    for a given channel via `estimated_channel_liquidity_range` (#1549).
+ * `ChannelManager::force_close_channel` has been renamed
+   `force_close_broadcasting_latest_txn` and a non-broadcasting companion has
```

Comment:
maybe this non-broadcasting companion deserves due mention `force_close_without_broadcasting_txn`, it's a nice companion too.